.github/workflows/code-formatting.yml

```
@@ -0,0 +1,23 @@
+name: Lint
+
+on: [push]
```

This only affects branches [in the repository](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-a-single-event). The following seems like it would also do pull requests:

```suggestion
on: [push, pull_request]
```